Stratis.Bitcoin/Configuration/TextFileConfiguration.cs

```diff
@@ -44,10 +40,11 @@ public TextFileConfiguration(string[] args)
             foreach (var arg in args)
```

Comment:
We could fix the violation of code style #10 here as a part of this PR. I.e. we are looking for 

```
foreach (string arg in args)
``` 

here.